sbncode/Calibration/TrackCaloSkimmer_module.cc

```diff
@@ -313,6 +315,9 @@ void sbn::TrackCaloSkimmer::analyze(art::Event const& e)
   for (art::Ptr<recob::PFParticle> p_pfp: PFParticleList) {
     const recob::PFParticle &pfp = *p_pfp;
 
+    if(p_pfp->PdgCode() == 11)
```

Comment:
Why is this being added in? I don't like having selection code inside the module itself. Selection code should go into one of the selection tools.

Reply:
That's fine then, this can stay